### ISSUE TITLE
Patient self-registration form isn't actually a form

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3984]
## Area or Component
Patient Experience - Self registration

## Quick Summary
The form is created entirely through `div` elements, not actually a `form` element. (The subcomponents of the form are more accurate elements like fieldset, legend, etc etc.)  

## Accessibility Violation
All page content should be contained by landmarks
https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome

This also potentially has implications for how screenreaders interact with the form and read the questions. We should do another functional test with AxeTools once the form component has been accurately created. (In particular, there were concerns with keyboard focus on the radio buttons that may be addressed with a new element).

## Action Requested
Put all the form questions for self-registration inside a `form` element.

The patient self-registration page is also missing a `main` element, which should be the parent element for the `form`.

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/app/patients/Components/PersonForm.tsx#L77-L77

Note: This is likely going to be solved at the `PersonForm` level, meaning we also need to check the in-app `Add/Edit Person` page since the component is shared.

**Comment by emmastephenson** [comment]
Note: going back over the guidance from Deque, it seems like the HTML `<form>` element is not always supported. Iceboxing this for now.
